### ISSUE TITLE
The accuracy of my current training is not up to the accuracy in your paper

### ISSUE THREAD

**Issue by LKELN** [bytedance/R2Former#8]
The accuracy of my current training is not up to the accuracy in your paper, I have trained it in full twice, the first time in A800 R@1: 87.4, R@5: 93.8, R@10: 94.5, R@20: 95.4, R@100: 97.0,the second time in RTX3090 R@1: 84.5, R@5: 91.5, R@10: 93.2, R@20. 93.8, R@100: 95.5
Is this the dataset you used?
![b4fa6c8c1a2ee263b6a5da88903f579](https://github.com/bytedance/R2Former/assets/109862533/57b2d016-a91d-42b4-8f62-5a1161aba80e)

**Comment by Jeff-Zilence** [comment]
Can you describe your experiment setting? For example, which script did you run? Did you modify anything in the code?

**Comment by LKELN** [comment]
I used the experiment setting you provided, I used the train_global_retrieval.sh first to get a global retrieval model for rerank resume. Then I use train_reranking.sh and I did not change anything.

**Comment by LKELN** [comment]
Except for the batchsize size, because I don't have that much memory

**Comment by Jeff-Zilence** [comment]
You should be able to get the same result if you are not using an extremely small batch size and follow all the instructions carefully. The learning rate must be tuned if you are using a very different batch size from our paper. There is no way for me to figure out the issue if the hyper-parameters (including package versions) and logs are not given. 
There are several steps that you should take to debug:
1. download the pretrained model and use test.sh. See if you can get R@1 around 89.7.
2. use train_global_retrieval.sh and see if you can get R@1 over 79.
3. use the first command of train_reranking.sh and see if you can get R@1 around 88.4.
4. use the second command of train_reranking.sh and see if you can get R@1 around 89.7.
5. If you want to train it end-to-end. You can directly run train_end_to_end.sh and the R@1 should be around 87.3.

**Comment by LKELN** [comment]
1. I can get R@1 around 89.7
2.I use train_global_retrieval.sh and get R@1 74.5
Maybe need to change the learn rate for train_global_retrieval.sh

, or may you provide a finetune model for global retrieval? And the msls_v2_deits.pth is no finetune model for global retrieval? [info.log](https://github.com/bytedance/R2Former/files/12644010/info.log)[info (1).log](https://github.com/bytedance/R2Former/files/12644017/info.1.log)

**Comment by Jeff-Zilence** [comment]
Please read the instructions carefully. The global retrieval model is provided as "msls_v2_deits.pth" on the ReadMe page. The global retrieval model follows the standard pipeline of the VG benchmark and there is no finetuning. Please read the paper for more details.

**Comment by LKELN** [comment]
Ok, Thanks for your reply!
